sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/ServiceBusDescribedType.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.models;
+
+import org.apache.qpid.proton.amqp.DescribedType;
+
+/**
+ * Use described type to send OffsetDatetime, Duration, URI etc. on the wire.
+ */
+public class ServiceBusDescribedType implements DescribedType {
```

Comment:
Is this class need to be public, or to be exposed to user? If not, move to implementation.